deployment/terraform_azure/azure_sql.tf

```diff
@@ -0,0 +1,45 @@
+resource "azurerm_sql_server" "eventuate_server" {
+  name                         = "${var.project}-ss-${var.env}-${random_id.random_suffix.hex}"
+  resource_group_name          = azurerm_resource_group.default.name
+  location                     = azurerm_resource_group.default.location
+  version                      = "12.0"
+  administrator_login          = var.sql_admin_user
+  administrator_login_password = var.sql_admin_password
+
+  tags = {
+    environment = var.env
+  }
+}
+
+resource "azurerm_sql_database" "eventuate" {
+  name                = "eventuate"
+  resource_group_name = azurerm_resource_group.default.name
+  location            = azurerm_resource_group.default.location
+  server_name         = azurerm_sql_server.eventuate_server.name
+  requested_service_objective_name = "S0"
+
+  tags = {
+    environment = var.env
+  }
+
+  provisioner "local-exec" {
+    command = <<EOF
+        wget --quiet https://raw.githubusercontent.com/eventuate-foundation/eventuate-common/0.10.0.RELEASE/mssql/1.setup.sql -O 1.setup.sql
```

Comment:
best to use  `curl https://raw.githubusercontent.com/eventuate-foundation/eventuate-common/0.10.0.RELEASE/mssql/1.setup.sql | sqlcmd ...` if you can